src/backend/aspen/api/conftest.py

```diff
@@ -124,14 +123,10 @@ async def override_get_db(
         await session.close()  # type: ignore
 
 
-async def override_get_auth_user(
+async def override_get_cookie_userid(
```

Comment:
Only override one auth method in our tests - this means that our magic link auth should work properly from pytest.